ods_ci/tasks/Resources/Provisioning/GPU/gpu_deploy.sh

```diff
@@ -40,9 +40,52 @@ function wait_until_gpu_pods_are_running() {
 
 }
 
+function rerun_accelerator_migration() {
+#As we are adding the GPUs after installing the operator, those GPUs are not discovered automatically.
```

Comment:
after instaling Nvidia operator you mean?

Reply:
I mean RHODS operator @bdattoma

Reply:
could you plz write it explicitly? 
btw, sounds like a bug

Reply:
Done